prow/config.yaml

```diff
@@ -708,6 +708,27 @@ postsubmits:
             privileged: true
         nodeSelector:
           cloud.google.com/gke-nodepool: new-cluster-pool
+    - name: istio-pilot-e2e-envoyv2-v1alpha3
+      agent: kubernetes
+      context: prow/istio-pilot-e2e-envoyv2-v1alpha3.sh
```

Comment:
please remove context, rerun_comands and trigger as those don t apply to postsubmit